<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWrite</class>
 <widget class="QDialog" name="DialogWrite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>327</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>202</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Write chip</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_Chip">
       <property name="text">
        <string>Chip type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="chip">
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_VPP">
       <property name="text">
        <string>Programming voltage VPP:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_d">
       <property name="text">
        <string>datasheet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="d_VPP">
       <property name="text">
        <string>Unkdown</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_s">
       <property name="text">
        <string>settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="s_VPP">
       <property name="text">
        <string>Unrdown</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="CEProgram">
       <property name="text">
        <string>CE Program Pulse Width:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QRadioButton" name="CEPulse_25">
       <property name="text">
        <string>25 us</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QRadioButton" name="CEPulse_50">
       <property name="text">
        <string>50 us</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QRadioButton" name="CEPulse_50000">
       <property name="text">
        <string>50 ms</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QRadioButton" name="CEPulse_100">
       <property name="text">
        <string>100 us</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QRadioButton" name="CEPulse_10000">
       <property name="text">
        <string>10 ms</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QRadioButton" name="CEPulse_500">
       <property name="text">
        <string>500 us</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QRadioButton" name="CEPulse_200">
       <property name="text">
        <string>200 us</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QRadioButton" name="CEPulse_5000">
       <property name="text">
        <string>5 ms</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QRadioButton" name="CEPulse_1000">
       <property name="text">
        <string>1 ms</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="WandV">
       <property name="text">
        <string>Run verification after writing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogWrite</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogWrite</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
